clients/spark-extensions-basetests/src/main/java/org/projectnessie/spark/extensions/AbstractNessieSparkSqlExtensionTest.java

```diff
@@ -628,7 +628,8 @@ void executeAndValidateCompaction(String branchName, String branchHash, String t
       throws NessieNotFoundException {
     List<Object[]> result = executeCompaction(tableName);
     // re-written files count is 2 and the added files count is 1
-    assertThat(result).hasSize(1).containsExactly(row(2, 1));
+    assertThat(result.get(0)[0]).isEqualTo(2);
+    assertThat(result.get(0)[1]).isEqualTo(1);
```

Comment:
I did try `.extracting()` but it became too messy/complex as `result.get(0)` can have a bigger array in the latest iceberg.

Reply:
nit: `.first()` might help. Also, would just dropping `hasSize(1)` and using `.containt(row(2, 1))` work?